subjects/devops/grades/README.md

```diff
@@ -0,0 +1,108 @@
+## grades
+
+### Instructions
+
+Create a script `grades.sh` which will ask the user for a specific number of students in order to evaluate them. After the number is set, the script will ask you to introduce the "names" and "grades" of the persons you wish to evaluate.
+
+The grades will have a range between 0 and 100 and will be numeric values only.
+
+According to each student grade you will have to return the following:
```

Comment:
The order here should be swapped, if you start by `< 50`, `>= 50`, `>= 70` and so on by logic a student having `100` should print all those messages: `>= 50`, `>= 70`, `>= 90`.
Instead we want to print only the `>= 90`, so the swapping.